src/stories/containers/transparency-report/transparency-mkr-vesting/transparency-mkr-vesting.tsx

```diff
@@ -60,51 +61,56 @@ export const TransparencyMkrVesting = (props: TransparencyMkrVestingProps) => {
       <Title isLight={isLight} marginBottom={24}>
         MKR Vesting Overview
       </Title>
-
-      <TotalFte isLight={isLight}>
-        <span>Total FTE</span>
-        <u>{FTEs}</u>
-      </TotalFte>
-
+      <ContainerPopover>
+        <CustomPopover
+          title={'Full-Time Equivalents'}
+          id={'popover-fulltime equivalent'}
+        >
+          <TotalFte isLight={isLight}>
+            <span>Total FTEs</span>
+            <u>{FTEs}</u>
+          </TotalFte>
+        </CustomPopover>
+      </ContainerPopover>
       {items.length - 1 <= 0
         ? (
-        <TransparencyEmptyTable />
+          <TransparencyEmptyTable />
           )
         : (
-        <>
-          <TableWrapper>
-            <InnerTable
-              headers={headers}
-              headersAlign={['left', 'right', 'right', 'right', 'left']}
-              headerStyles={[{}, {}, {}, {}, { paddingLeft: '38px' }]}
-              items={items}
-              minWidth={200}
-              headerWidths={['200px', '210px', '210px', '210px', '354px']}
-              style={{ marginBottom: '32px' }}
-            />
-          </TableWrapper>
-
-          <CardsWrapper>
-            {items.map((item) => (
-              <TransparencyCard
-                header={item[0]}
-                headers={headers.slice(1)}
-                items={item.slice(1)}
+          <>
+            <TableWrapper>
+              <InnerTable
+                headers={headers}
+                headersAlign={['left', 'right', 'right', 'right', 'left']}
+                headerStyles={[{}, {}, {}, {}, { paddingLeft: '38px' }]}
+                items={items}
+                minWidth={200}
+                headerWidths={['200px', '210px', '210px', '210px', '354px']}
+                style={{ marginBottom: '32px' }}
               />
-            ))}
-          </CardsWrapper>
-
-          <Text isLight={isLight}>
-            This Overview is based on MIP40c3-SP17, SES’ MKR Incentive Proposal.
-          </Text>
-
-          <Text isLight={isLight} style={{ marginBottom: '90px' }}>
-            The Difference column indicates any changes in the MKR vesting
-            amounts compared to last month, with the Reason(s) column indicating
-            why the amounts changed. Reasons may include: New hires, FTE
-            changes, Promotions, or Terminations.
-          </Text>
-        </>
+            </TableWrapper>
+
+            <CardsWrapper>
+              {items.map((item) => (
+                <TransparencyCard
+                  header={item[0]}
+                  headers={headers.slice(1)}
+                  items={item.slice(1)}
+                />
+              ))}
+            </CardsWrapper>
+
+            <Text isLight={isLight}>
+              This Overview is based on MIP40c3-SP17, SES’ MKR Incentive Proposal.
```

Comment:
There is a `’` character after `SES’`, is it ok?

Reply:
yes,  there is not problem with that